### PR TITLE
Allow caller to handle case where element is not found

### PR DESCRIPTION
**Pull request by lottspot**

### CODE REVIEW

File: src/js/core.js [hunk 1]
```diff
@@ -45,13 +45,12 @@ module.exports.getUrlParameter = (name) => {
...
-  let template;
-  try {
-    template = document.getElementById(id).innerHTML;
-  } catch (e) {
-    template = '';
+  let template = document.getElementById(id);
+  if (template) {
+    return parseHTML(Mustache.render(template.innerHTML, context));
+  } else {
+    return null;
```
Comment (modpluz) [inline src/js/core.js:52]: Is there a specific reason to `return null` here? Wouldn't just `return` suffice?

Comment (lottspot) [reply src/js/core.js:52]: A fair question. The idea behind this method is that I want it to mimic the behavior of `document.getElementById` as closely as possible; pass it an element ID, and get a `NodeList` back (I realize this method actually returns a `DocumentFragment`, but same idea...). So the idea in returning `null` is to mimic the behavior of `getElementById`, which itself returns null if an element with the provided id can't be found.

Comment (modpluz) [reply src/js/core.js:52]: I see your point.

However, `renderTemplate()` returns the result of `parseHTML()` (which is `DocumentFragment` as mentioned above and as per [MDN](https://developer.mozilla.org/en-US/docs/Web/API/Range/createContextualFragment)) and not that of `document.getElementById()`.

**Review by modpluz — APPROVED** [review]
Look out, there's a merge ahead!

File: src/js/core.js (hunk 1 above)
Comment (lottspot) [reply src/js/core.js:52]: > I want it to mimic the behavior of `document.getElementById` **as closely as possible**

I think I conceded that it was not perfect mimicry of `getElementById` 😉

**Pull request merged**